Code/Framework/AzToolsFramework/AzToolsFramework/API/ViewportEditorModeTrackerNotificationBus.h

```diff
@@ -23,11 +23,11 @@ namespace AzToolsFramework
         Pick
     };
 
-    //! Viewport identifier and other relevant viewport data.
+    //! Viewport editor mode tracker identifier and other relevant data.
     struct ViewportEditorModeInfo
```

Comment:
question: Should this be renamed to `ViewportEditorModeTracker` (or `ViewportEditorModeTrackerInfo`) just going from the comment update?

Reply:
fixed 👍